pkg/chartvalues/flannel_operator_test.go

```diff
@@ -0,0 +1,159 @@
+package chartvalues
+
+import (
+	"testing"
+
+	"github.com/giantswarm/e2etemplates/internal/rendertest"
+)
+
+func newFlannelOperatorConfigFromFilled(modifyFunc func(*FlannelOperatorConfig)) FlannelOperatorConfig {
+	c := FlannelOperatorConfig{
+		ClusterName: "test-cluster",
+		ClusterRole: FlannelOperatorClusterRole{
+			BindingName: "flannel-operator",
+			Name:        "flannel-operator",
+		},
+		ClusterRolePSP: FlannelOperatorClusterRole{
+			BindingName: "flannel-operator-psp",
+			Name:        "flannel-operator-psp",
+		},
+		RegistryPullSecret: "test-registry-pull-secret",
+	}
+
+	modifyFunc(&c)
```

Comment:
Just FMI what is this doing again?

Reply:
You get the fully qualified template from above, the modifyFunc just changes e.g. individual values to just set them differently for a specific test.

That way you don't have to restate the entire foo but can instead just overwrite with the modifyFunc for  your test.

Reply:
https://github.com/giantswarm/e2etemplates/pull/29/files#diff-b47b8619de1758c2f8de9af90adf8907R97